recipes-extended/xen/xen-common.inc

```diff
@@ -1,4 +1,4 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+FILESEXTRAPATHS_prepend := "${THISDIR}/../../../meta-virtualization/recipes-extended/xen/files:${THISDIR}/files:"
```

Comment:
> If that is specific to `xen-ocaml-libs`, it probably belongs in that recipe.

It could go there but the intent was to make sure in an obvious way that the search path for the SRC_URI is the same for each of the OpenXT Xen recipes to avoid surprises. Do you need it to be moved or does this justify it being there?

> Why are the OCaml pieces requiring meta-virt patches if meta-virt does not support OCaml btw? Headers I presume? (introduced by [f439396](https://github.com/OpenXT/xenclient-oe/commit/f439396ec678bec79daa9c98b7ff80ddecfaee8e), which seems to point at it being ocaml specific)

IIRC it's build system stuff: there are python script fixes needed to make the build happy since python2 is not used in the build now.

Reply:
This isn't needed with the current version of meta-virtualization and a ported xenclient-oe to use it.